core/src/main/java/org/apache/calcite/sql/dialect/SparkSqlDialect.java

```diff
@@ -744,6 +745,12 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
       writer.literal(creteDateTimeFormatSqlCharLiteral(call.operand(1).toString()).toString());
       writer.endFunCall(dateDiffFrame);
       break;
+    case "PARSE_TIMESTAMP":
+      formatString = creteDateTimeFormatSqlCharLiteral(call.operand(0).toString());
```

Comment:
Please fix the typo in method name. Change it to **createDateTimeFormatSqlCharLiteral**.

Reply:
Rename method name as suggested.